webcompat/static/js/lib/homepage.js

```diff
@@ -32,6 +32,7 @@ function HomePage() {
   this.setUpEvents = function() {
     this.searchHandler();
     this.dropDownHandler();
+    this.navbarHandler();
```

Comment:
I guess it feels a little weird to put this navbar controller code in homepage.js, but maybe it's more awkward we include homepage.js on every page. 😄 

We should probably file a followup bug to change that, and have something like navbar.js with fancy code-splitting. I imagine we'll move to a more modern modular design this year.

Reply:
Indeed. I needed some time to find the "correct" place to add this :)